pkg/performanceprofile/controller/performanceprofile/components/utils.go

```diff
@@ -79,11 +79,12 @@ func CPUListToMaskList(cpulist string) (hexMask string, err error) {
 type CPULists struct {
 	reserved cpuset.CPUSet
 	isolated cpuset.CPUSet
+	offlined cpuset.CPUSet
 }
 
 // Intersect returns cpu ids found in both the provided cpuLists, if any
-func (c *CPULists) Intersect() []int {
-	commonSet := c.reserved.Intersection(c.isolated)
+func (c *CPULists) Intersect(firstSet cpuset.CPUSet, secondSet cpuset.CPUSet) []int {
```

Comment:
minor/not blocking: this method doesn't use fields of `CPULists` anymore so it should be a free function.
And maybe, being it a two-liner, could be inlined in the calling sites.

Reply:
Fix. Thanks again